doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -27,7 +27,7 @@ The specifications defined in this chapter will be detailed with unique
 identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
 `ra2.ch.001` for the first requirement in the Kubernetes Node section.  These
 specifications will then be used as requirements input for the Kubernetes
-Reference Implementation and any Vendor or Community Implementations.
+Reference Implementation and any vendor or community implementations.
```

Comment:
Not sure about this one; I thought VI / CI were capitalisable just the same as RI?

Reply:
Thanks for the comment! Other mentions of "Vendor Implementation" seems to refer to the chapter Vendor Implementation within the documentation. In this case, however, I think it refers to actual vendor implementations and therefore it should not be capitalized.